### DISCUSSION TITLE
replacement for get-poetry.py

### DISCUSSION THREAD

**Discussion by aringa** [python-poetry/poetry#8605] [General]
Hi there 

[full-stack-fastapi-postgresql](https://github.com/tiangolo/full-stack-fastapi-postgresql) fails to build because it cannot download

https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py

The file doesn't exist anymore in master.

If it has been renamed, would you kindly point me to the renamed file.

**Comment by dimbleby** [comment]
please read the poetry docs to learn about the supported ways of installing poetry

**Comment by aringa** [comment]
I fixed [full-stack-fastapi-postgresql](https://github.com/tiangolo/full-stack-fastapi-postgresql) to point to tag 1.3.2 and the issue was resolved.

**Reply by dimbleby** [reply]
it would have been better to read the docs and use a recent version of poetry, but whatever
